tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{36,37,38,39}
+envlist = py{36,37,38,39,310,311}
```

Comment:
I believe we also need to update the list of python versions here so that they are tested on github actions
https://github.com/openlawlibrary/pygls/blob/4772250f8ede25aee8e457dae5bca394aeccdd2f/.github/workflows/ci.yml#L16

Reply:
Thank you @perrinjerome for the feedback. I tried to fixed the issues. Tox tests run fine with local Pyenv versions, and at least [esbonio LS](https://github.com/swyddfa/esbonio) (based on pygls) seems to be functioning correctly with the local installation of the PR.